### PR TITLE
add knwon issues for query template

### PR DESCRIPTION
**Pull request by Chentong-Xiang**
@Lauragra we don't have the plan to support query template for other entity types, so I put the explain into the description part. And is it okay that we proceed with this PR for now? In the future, I will create PR in the new environment you suggested, instead of the mirror one.

### CODE REVIEW

**Comment by Lauragra** [comment]
@Chentong-Xiang , please see our guidance for documenting Known issues, here: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211754/Known-issues

Also, please follow the Get started steps to set yourself up to contribute to the docs, and in the future open PRs in our contributors repo, here: https://github.com/microsoftgraph/microsoft-graph-docs/pulls

This repo is a mirror for public contributions only. Thanks.

File: concepts/search-concept-query-template.md [hunk 1]
```diff
@@ -134,6 +134,10 @@ Content-type: application/json
 }
 ```
 
+## Known issues
+
+- Query templates are supported only for SharePoint, OneDrive, or external items (i.e. site, drive, driveItem, list, listItem and externalItem). They are not supported for **message**, **chatMessage**, **event**, **person**, **bookmark**, **acronym** or **qna** types.
```
Comment (njerigrevious) [inline concepts/search-concept-query-template.md:139]: Query templates ARE supported for "message" - so let's update to reflect that: image

Query templates ARE supported for "event": 
image

seems like it's supported for "person" too: 
image

seems like "bookmark" too: 
image

seems like "acronym" too: 
image

please double check qna and chatMessage as well to see if it's supported. 

not sure if this means if it's supported or not for externalItem
image

**Review by njerigrevious — CHANGES_REQUESTED** [review]
Thanks for starting this PR! Please check all entityTypes to make sure we know if each is supported or not for queryTemplate.

**Comment by Chentong-Xiang** [comment]
> Thanks for starting this PR! Please check all entityTypes to make sure we know if each is supported or not for queryTemplate.

Thank you for your effort! I understand your point. In the code at the graph level, there are no restrictions on the "query template" field for any entity type; any query template can be passed down to the lower-level 3s request query template. However, it's important to note that 3s may not necessarily process query templates for all entity types. Considering the purpose of query templates is to support KQL filtering, we may need to set KQL in query template and to check if the returned results meet the KQL filter criteria (See the screenshot, I asked for emails from yiwenwang, but got your emails back). I tested all entityTypes in this way, so file and connector are good.
image

Additionally, for now, 3S does not have plans to support queryTemplate in other entity types other than "file" and "connector." Based on Lauragra's [feedback](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211754/Known-issues), is it better to put this update in the description?

File: concepts/search-concept-query-template.md (hunk 1 above)
Comment (Chentong-Xiang) [reply concepts/search-concept-query-template.md:139]: Thank you for your effort! I understand your point. In the code at the graph level, there are no restrictions on the "query template" field for any entity type; any query template can be passed down to the lower-level 3s request query template. However, it's important to note that 3s may not necessarily process query templates for all entity types. Considering the purpose of query templates is to support KQL filtering, we may need to set KQL in query template and to check if the returned results meet the KQL filter criteria (See the screenshot, I asked for emails from yiwenwang, but got your emails back). I tested all entityTypes in this way, so file and connector are good.
Additionally, for now, 3S does not have plans to support queryTemplate in other entity types other than "file" and "connector." Based on Lauragra's [feedback](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211754/Known-issues), is it better to put this update in the description?

Comment (njerigrevious) [reply concepts/search-concept-query-template.md:139]: Got it - thanks for clarifying @Chentong-Xiang! I agree with the current PR that we should put this in both the "known issues" and also put the information in the description. I'll approve this PR for now, but let's make sure the description is updated as well. Thanks!

**Review by njerigrevious — APPROVED** [review]
Thanks for clarifying!

**Pull request merged**